lib/Value/Matrix.pm

```diff
@@ -837,29 +837,155 @@ sub twiddle {
 	return $self->make(@coords);
 }
 
-=head3 C<transpose>
+=head3 C<slice>
 
-Take the transpose of a matrix.
+Apply this to a degree n Matrix, passing (m, k), and produce the degree (n-1) Matrix defined by
+taking all entries whose position has mth index with value k. For example if C<$M> is a 4x5x6
+Matrix, then m can be 1, 2, or 3. If m is 2, then k can be 1, 2, 3, 4, or  5. C<$M-<gt>slice(2,3)>
```

Comment:
The code at the end of this can be made with C<<$M->slice(2,3)>>, a bit easier than the <gt> notation.

Reply:
That actually needs to be `C<< $M->slice(2, 3) >>`.  There must be spaces after `<<` and before `>>`.

Reply:
Good point.